### PR TITLE
[PM-12011] Fix Unlock with biometrics on the Safari browser extension

### PR DESCRIPTION
**Pull request by Thomas-Avery**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-12011

## 📔 Objective

The purpose of this PR is to fix Unlock with biometrics on the Safari browser extension.

The `SafariWebExtensionHandler` needs to support the `biometricUnlockAvailable` command added in https://github.com/bitwarden/clients/pull/10374.

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/browser/src/safari/safari/SafariWebExtensionHandler.swift [hunk 1]
```diff
@@ -194,6 +205,21 @@ func jsonDeserialize<T: Decodable>(json: String?) -> T? {
...
+extension LAContext {
+    func isBiometricsAvailable() -> Bool {
+        var error: NSError?
+
+        self.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: &error)
+
+        if let e = error, e.code != kLAErrorBiometryLockout {
+            return false;
+        }
+        else {
```
Comment (quexten) [inline apps/browser/src/safari/safari/SafariWebExtensionHandler.swift:217]:
```suggestion
        } else {
```
:pick: nit-pick: looking at the style of the rest of the file, I believe this formatting is more in line with the rest.

**Review by quexten — APPROVED** [review]
Changes look good, nice work!

**Review by mzieniukbw — APPROVED** [review]

File: apps/browser/src/safari/safari/SafariWebExtensionHandler.swift (hunk 1 above)
Comment (Thomas-Avery) [reply apps/browser/src/safari/safari/SafariWebExtensionHandler.swift]: Updated in https://github.com/bitwarden/clients/pull/11040/commits/9f8983a609c17ab926a551b154204a364dabc2ef

**Review by quexten — APPROVED** [review]
Nice work!

**Pull request merged**